Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#*.pyc\r\n#.vscode\r\n/build\r\n/colmap-x64-windows-cuda\r\n/dist\r\n/output\r\n/viewers\r\n\r\n/*.png\r\n/*.jpg\r\n/*.spec\r\n/*.ui\r\n/cdx.txt\r\n#/test.py\r\n#/test1.py\r\n#/test2.py\r\n#/test3_.py\r\n#/test4_look_line.py\r\n#/test4_look_points.py\r\n\r\n\r\n#diff_rasterization/diff_rast.egg-info\r\n#diff_rasterization/dist\r\n#tensorboard_3d\r\n#screenshots
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 9544f3da89e6a5a7898dad8c9762d2b1d0a9e352)
+++ b/.gitignore	(date 1751451090935)
@@ -1,5 +1,4 @@
-#*.pyc
-#.vscode
+
 /build
 /colmap-x64-windows-cuda
 /dist
@@ -17,9 +16,3 @@
 #/test3_.py
 #/test4_look_line.py
 #/test4_look_points.py
-
-
-#diff_rasterization/diff_rast.egg-info
-#diff_rasterization/dist
-#tensorboard_3d
-#screenshots
\ No newline at end of file
